Simplenote/src/main/java/com/automattic/simplenote/WordPressDialogFragment.java

```diff
@@ -102,8 +103,9 @@ public void onClick(View view) {
 
                 ClipboardManager clipboard = (ClipboardManager) getActivity().getSystemService(Context.CLIPBOARD_SERVICE);
                 if (clipboard != null) {
-                    ClipData clip = ClipData.newPlainText("Simplenote", mPostUrlTextView.getText());
+                    ClipData clip = ClipData.newPlainText("Simplenote", mPostUrl);
                     clipboard.setPrimaryClip(clip);
+                    Toast.makeText(requireContext(), R.string.copied_to_clipboard, Toast.LENGTH_SHORT).show();
```

Comment:
Noice

Reply:
I thought a little confirmation might be nice.